src/vm/runners/cairo_runner.rs

```diff
@@ -303,17 +303,32 @@ impl CairoRunner {
         Ok(())
     }
 
+    /// Mark a memory address as accesed.
+    pub fn mark_as_accessed(
+        &mut self,
+        address: Relocatable,
+        size: usize,
+    ) -> Result<(), VirtualMachineError> {
+        let accessed_addressess = match &mut self.accessed_addresses {
+            Some(x) => x,
+            None => return Err(VirtualMachineError::RunNotFinished),
+        };
```

Comment:
This looks like a candidate for:
```suggestion
        let accessed_addressess = self.accessed_addresses.ok_or(VirtualMachineError::RunNotFinished)?;
```